pkgs/applications/science/biology/delly/default.nix

```diff
@@ -2,15 +2,17 @@
 
 stdenv.mkDerivation rec {
   pname = "delly";
-  version = "0.8.1";
+  version = "0.8.2";
 
   src = fetchFromGitHub {
       owner = "dellytools";
       repo = pname;
       rev = "v${version}";
-      sha256 = "18gm86j1g1k4z1cjv2m5v9rsl1xqs2w3dhwcsnzx2mhkrvmlc4i1";
+      sha256 = "14bkmixz7737xj192ww96s3a20zc7xs7r04db8avw3ggi3i1s1cs";
   };
 
+  patches = [ ./htslib.patch ];
```

Comment:
Could you please use `fetchpatch` to fetch from https://github.com/dellytools/delly/commit/0e5c710b0c5ea790bb39699d4cbd49cf4fb86f14.diff

Reply:
The patch has been modified slightly to remove the subproject commit hash update on lines 50-56. How should this be handled?

Reply:
`fetchpatch` has an option to exclude certain files
https://github.com/NixOS/nixpkgs/blob/0566b5ce19fabe3c70ab1f56415dd8a019fa0aa8/pkgs/build-support/fetchpatch/default.nix#L12

Don't forget to update the hash every time you change the parameters or fixed-output derivation won't be rebuilt.

Reply:
Neat, thanks!